stdlib/InteractiveUtils/src/codeview.jl

```diff
@@ -137,7 +137,7 @@ characteristics of a particular type is an implementation detail of the compiler
 concern, so some types may be colored red even if they do not impact performance.
 Small unions of concrete types are usually not a concern, so these are highlighted in yellow.
 
-Keyword argument `debuginfo` may be one of `:source` or `:none` (default), to specify the verbosity of code comments.
+Keyword argument `debuginfo` may be one of `:source` or `:none` (equivalently, `:default`), to specify the verbosity of code comments.
```

Comment:
Actually I think the intention of the original text was to convey that `:none` is the default (or rather, the "default default"...) -- `:default` is an implementation detail here

The new text is IMHO worse because `:default` is *not* equivalent to `:none`. To quote the end of `doc/src/base/reflection.md`:

> Possible values for `debuginfo` are: `:none`, `:source`, and `:default`.
> Per default debug information is not printed, but that can be changed
> by setting `Base.IRShow.default_debuginfo[] = :source`.

Reply:
I have force-pushed a new version that takes your comment into account.

Two comments about `Base.IRShow.default_debuginfo`:
- It seems to me that this symbol is not public.
- Contrary to what it suggested in the documentation, it does not apply to `code_native`.

Reply:
Indeed I also don't think `Base.IRShow.default_debuginfo` is public and hence would not want to reference it in a docstring like that.

I also don't see how mentioning the value `:default` is helpful for the user -- to me it seems to be an implementation detail, not something to be relied on. So I really fail to grasp in how far this change improves anything; I definitely don't see how it "fixes" something as the current text seems fine to me, if perhaps not as precise as it should be?

Reply:
The value `:default` is already mentioned: The beginning of the docstring for `code_warntype` says
```
    code_warntype([io::IO], f, types; debuginfo=:default)
```
So I think it's confusing when later in the docstring it is stated that
```
Keyword argument `debuginfo` may be one of `:source` or `:none`
```
without mentioning `:default`.